packages/reports/tests/unit/consumers/request/sort-test.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 import { moduleFor, test } from 'ember-qunit';
 import { RequestActions } from 'navi-reports/services/request-action-dispatcher';
 
-const { get, getOwner } = Ember;
+const { getOwner } = Ember;
```

Comment:
We're slowly moving towards using the new ember import styles so could we change this to
`import { getOwner } from '@ember/application'`
and then get rid of the `import Ember from 'ember'`?